hello-kubernetes/README.md

```diff
@@ -40,8 +40,8 @@ $ dapr init --kubernetes
 Dapr can use a number of different state stores (Redis, CosmosDB, DynamoDB, Cassandra, etc) to persist and retrieve state. For this demo, we'll use Redis.
 
 1. Follow [these steps](https://docs.dapr.io/getting-started/configure-redis/) to create a Redis store.
-2. Once your store is created, add the keys to the `redis.yaml` file in the `deploy` directory. 
-    > **Note:** the `redis.yaml` file provided in this quickstart takes plain text secrets. In a production-grade application, follow [secret management](https://docs.dapr.io/developing-applications/building-blocks/secrets/) instructions to securely manage your secrets.
+2. Once your store is created, add the keys to the `redis.yaml` file in the `deploy` directory.
+    > **Note:** the `redis.yaml` file provided in this quickstart will work securely out-of-the-box with a Redis installed with `helm install bitnami/redis`. If you have your own Redis setup, replace the `redisHost` value  with your own Redis master address, and the redisPassword with your own Secret.
```

Comment:
Add a comment to learn more [here](https://docs.dapr.io/operations/components/component-secrets/)

Reply:
Done